dev/breeze/doc/ci/01_ci_environment.md

```diff
@@ -0,0 +1,132 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+
+- [CI Environment](#ci-environment)
+  - [GitHub Actions runs](#github-actions-runs)
+  - [Container Registry used as cache](#container-registry-used-as-cache)
+  - [Authentication in GitHub Registry](#authentication-in-github-registry)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# CI Environment
+
+Continuous Integration is important component of making Apache Airflow
+robust and stable. We are running a lot of tests for every pull request,
+for main and v2-\*-test branches and regularly as scheduled jobs.
+
+Our execution environment for CI is [GitHub
+Actions](https://github.com/features/actions). GitHub Actions (GA) are
+very well integrated with GitHub code and Workflow and it has evolved
+fast in 2019/202 to become a fully-fledged CI environment, easy to use
```

Comment:
2019/2020* to become

Reply:
removed it altogether and updated. It's prehistory, not relevant any more.